techmatters/flex-plugins
# [Bug Fix] Warm Transfer - Caller is not always placed on hold

## Description
This PR adds small changes around transfer and conferencing logic to prevent weird behaviors encountered when performing warm voice transfers from one counsellor to another.
For a full description of the bug, check the ticket linked below.

### Checklist
- [x] [Corresponding issue has been opened](https://tech-matters.atlassian.net/browse/CHI-2025)
- [n/a] Feature flags added
- [n/a] Strings are localized
- [n/a] Tested for chat contacts

### Verification steps
- Start two new Flex instances using this branch (use Incognito tab to open a second instance).
- Start a voice call, accept with one Flex instance.
- Start a warm transfer to the other Flex instance.
- Confirm that the bug described in the bug ticket is not happening anymore.
